modules/vaos/app/controllers/vaos/limits_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
```

Comment:
Have we found that this is actually required? I would have thought it would be loaded automatically as part of the engine.